sinks/api/http/openapi.yaml

```diff
@@ -350,6 +364,39 @@ components:
             remote_host: my.prometheus-host.com
             username: dbuser
           description: Object representing backend specific configuration information
+    SinkCreateReqV2Schema:
+      type: object
+      required:
+        - name
+        - backend
+        - config
+      properties:
+        name:
+          type: string
+          description: A unique name label
+          example: my-prom-sink
+        description:
+          type: string
+          description: User description of this Sink
+          example: An example prometheus sink
+        tags:
+          type: object
+          description: User defined key/values for organization and searching
+          example:
+            cloud: aws
+        backend:
+          type: string
+          example: prometheus
+          description: The sink backend to use. Must match a backend from /features/sinks. Cannot change once created.
+        format:
+          type: string
+          enum:
+            - yaml
+            - json
+          description: Format that the config will be passed on
+        config:
```

Comment:
this is missing config_data isn't it?

Reply:
good catch, I will change in my next PR!

Reply:
Commit with changes
https://github.com/orb-community/orb/commit/27add81715582d191df00103ad74607cef7d68e0